package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "publish",
+  "version": "0.1.0-dev",
+  "description": "A CLI tool for publishing a node package to npm",
+  "main": "publish.js",
+  "engines": {
+    "node": ">=0.12.0"
+  },
+  "scripts": {
+    "test": "node tests/publishTests.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jobara/publish.git"
+  },
+  "keywords": [
+    "npm",
+    "publish",
+    "node",
+    "module",
+    "CLI"
+  ],
+  "author": "Fluid Project",
+  "license": "BSD-3-Clause",
+  "bugs": {
+    "url": "https://github.com/jobara/publish/issues"
+  },
+  "homepage": "http://fluidproject.org",
+  "dependencies": {
+    "es6-template-strings": "2.0.0"
+  },
+  "devDependencies": {
+    "grunt": "^0.4.5",
+    "grunt-contrib-jshint": "~0.9.0",
+    "grunt-jsonlint": "1.0.4",
+    "sinon": "^1.17.1"
+  },
+  "defaultOptions": {
+    "changes": "git status -s -uno",
```

Comment:
I think for clarity all these options which are commands should be distinguished - either i) through a suffix like "Command" or "Cmd" or else being nested in an options area named "commands"